package.json

```diff
@@ -56,7 +56,7 @@
     "eslint-plugin-react": "^4.2.3",
     "json-loader": "^0.5.4",
     "mocha": "^2.3.4",
-    "opentracing": "^0.10.6",
+    "opentracing": "^0.11.0",
```

Comment:
(I won't merge this PR or push a new npm version until the OT-js PR goes in: https://github.com/opentracing/opentracing-javascript/pull/41)